readme.md

```diff
@@ -560,6 +560,25 @@ interface Attestation {
 ```
 
 
+#### 6. Attester rejects attestation
+
+In case the attester does not approve the attestation request, no information about this appears on the blockchain.
+The extension can only get this information directly from the attester. 
+This could be useful to remove the corresponding credential from the extension.
```

Comment:
This could be controversial since removing things from the extension should always happen with the users consent and out of his free will.

```suggestion
This could be useful to mark the credential as rejected or to prompt the user to remove the corresponding credential from the extension.
```

Reply:
It’s inconceivable to me that someone would do that without user confirmation :shrug:

Reply:
How do you like the new wording?